### PR TITLE
Add v1.15.0 release blog post

### PR DESCRIPTION
**Pull request by pmuens**
**Todos:**

- [x] Write content for each headline / feature
- [x] Update contributors list

### CODE REVIEW

**Comment by pmuens** [comment]
Initial draft is done. Will do the last minor changes tomorrow once the release is in progress / done.

/cc @DavidWells @worldsoup

**Comment by pmuens** [comment]
**Just a quick update:**

The autocomplete PR (https://github.com/serverless/serverless/pull/3702) is not merged yet so this blog post might need an quick update before releasing.

**Comment by worldsoup** [comment]
schedule(06/09/2017 5:00 am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **06/09/2017 at 5:00am (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/189#issuecomment-307144514)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

File: posts/2017-06-09-serverless-v1.15.0.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+Serverless v1.15 ships with support to validate the CloudFormation template before kicking off the deployment phase.
+
+This feature could heavily improve the feedback loop because developers don't have to wait for a failing deployment to see that their CloudFormation template might include some (trivial) errors such as cyclic dependencies.
+
+This feature is enabled by default and is especially helpful when you `serverless deploy` your own deployment packages which might be generated or modified by another plugin / tool.
+
+### `serverless.json` support
+
+Usually you write your services definition in you `serverless.yml` or `serverless.yaml` file.
```
Comment (j0k3r) [inline posts/2017-06-09-serverless-v1.15.0.md]:
```diff
-in you
+in your
```

Or maybe just remove the `you`.

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**